src/Native/Time.js

```diff
@@ -12,37 +12,36 @@ Elm.Native.Time.make = function(elm) {
 
   function fpsWhen(desiredFPS, isOn) {
```

Comment:
I find the new version of this function very very confusing. At least it should not have 170+ character lines, but the ideal version might actually move `wasOn` and `timeoutID` into a signal so that it all gets picked up by Elm Reactor?

The same style stuff with `{}` applies here as well with the function and object. Prefer to bring things down a line.